cli/state.go

```diff
@@ -1028,7 +1054,21 @@ var compStateMsg = `
   <table>
    <tr><th>Name</th><th>Total/Compute/Storage</th><th>Time Taken</th><th>Location</th></tr>
    {{range .GasCharges}}
-    <tr><td>{{.Name}}</td><td>{{.TotalGas}}/{{.ComputeGas}}/{{.StorageGas}}</td><td>{{.TimeTaken}}</td><td>{{.Location}}</td></tr>
+    <tr><td>{{.Name}}</td><td>{{.TotalGas}}/{{.ComputeGas}}/{{.StorageGas}}</td><td>{{.TimeTaken}}</td>
+	<td>
+	{{ range $index, $ele := .Location }}
+	 {{- if $index }}|&#8203;{{end -}}
+	 {{- if .Show -}}
+	 {{- if .Important }}<b>{{end -}}
+	 {{- . -}}
+	 {{- if .Important }}</b>{{end -}}
+     {{- else -}}
+     <label class="ellipsis-toggle"><input type="checkbox" /><span class="ellipsis">...</span>
+	 {{- "" -}}
+	 <span class="ellipsis-content">{{- . -}}</span></label>
+	 {{- end -}}
+	{{end}}
+	</td></tr>
```

Comment:
How about we make this into a `<details>` block, with summary being just the first Important frame, and expand into a nice multi-line trace?

Reply:
There isn't always a trace marked important but I can try.